conf/routes

```diff
@@ -3,6 +3,10 @@ GET     /                                 controllers.App.index
 GET     /api/originatingSystem/:system    controllers.App.getStartedIn(system: String)
 GET     /api/commissioningDesks           controllers.App.getCommissioningDeskList
 
+# endpoints for posting data from other apps
+OPTIONS /api/metric/*url                  controllers.App.allowCORSAccess(methods = "PUT, POST, DELETE", url: String)
+POST    /api/metric/save                  controllers.App.saveMetric()
```

Comment:
This endpoint enables us to save metrics data all of which we get from our digital systems. I was thinking that this could be renamed to indicate this and then in future work we can have an endpoint for saving metrics from incopy. Perhaps something like `/api/metric/digital/save`?

Reply:
What about `/api/metric/digital` then? We already know it's a save because we're doing a `POST`.